src/components/volume-viewer/VolumeViewerWrapper.jsx

```diff
@@ -0,0 +1,17 @@
+import Controls from "../controls/Controls";
+import VolumeViewer from "./VolumeViewer";
+
+import { useSelectorContext } from "../../context/selector-context";
+
+export default function VolumeViewerWrapper(props) {
+  const { state } = useSelectorContext();
+  return (
+    <div>
+      <Controls
+        sidebarVisible={props.sidebarVisible}
+        setSidebarVisible={props.setSidebarVisible}
+      />
+      <VolumeViewer volume={state} />
```

Comment:
Why isn't the `VolumeViewer` component grabbing the state on its own? Is that because it's a class based component?